ckanext/data_qld/resource_format_openness_scores.json

```diff
@@ -0,0 +1,84 @@
+[
+  ["_comment",
+   "JSON field order as follows:",
+   ["Format", "Score"],
+   "where:",
+   " * Format - the short name for it, corresponding to the name in ckan's resource_formats.json.",
```

Comment:
This wording is very informal.

Reply:
@ThrawnCA 

a) it's a comment
b) it's from the origin repository: https://github.com/ckan/ckanext-qa/blob/master/ckanext/qa/resource_format_openness_scores.json